pkgs/tools/misc/woeusb-ng/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, python3Packages, fetchFromGitHub, p7zip, parted, grub2 }:
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "woeusb-ng";
+  version = "0.2.10";
+
+  propagatedBuildInputs = [ p7zip parted grub2 termcolor wxPython_4_0 six ];
+
+  src = fetchFromGitHub {
+    owner = "WoeUSB";
+    repo = "WoeUSB-ng";
+    rev = "v${version}";
+    sha256 = "sha256-Nsdk3SMRzj1fqLrp5Na5V3rRDMcIReL8uDb8K2GQNWI=";
+  };
+
+  postInstall = ''
+    # the maintainer suggested splitting the package into two packages, therefore the gui is removed
+    rm $out/bin/woeusbgui
+  '';
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "WoeUSB-ng is a simple tool that enable you to create your own usb stick windows installer from an iso image or a real DVD. This is a rewrite of original WoeUSB.";
```

Comment:
Too long description. This field should be short (and not duplicate the package name). Use an additional longDescription field for long descriptions.

Reply:
better now?

Reply:
Still to long. Please see https://nixos.org/manual/nixpkgs/stable/#var-meta-description.

Reply:
I think I would have used this (copied right from the homepage): `A Linux program to create a Windows USB stick installer from a real Windows DVD or image`.